service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerAboutToSubmitTest.java

```diff
@@ -0,0 +1,238 @@
+
+        given(documentGeneratorService.generateDocmosisDocument(any(), any())).willReturn(docmosisDocument);
+    }
+
+    @Test
+    void shouldPopulateHiddenCCDFieldsInStandardDirectionOrderToPersistData() {
+        given(uploadDocumentService.uploadPDF(PDF, SEALED_ORDER_FILE_NAME))
+            .willReturn(document);
+
+        UUID uuid = UUID.randomUUID();
+
+        List<Element<Direction>> fullyPopulatedDirection = List.of(
+            element(uuid, Direction.builder()
+                .directionType("Identify alternative carers")
+                .directionText("Contact the parents to make sure there is a complete family tree showing family"
+                    + " members who could be alternative carers.")
+                .assignee(LOCAL_AUTHORITY)
+                .directionRemovable("Yes")
+                .readOnly("Yes")
+                .build()));
+
+        List<Element<Direction>> directionWithShowHideValuesRemoved = buildDirectionWithShowHideValuesRemoved(uuid);
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .data(createCaseDataMap(directionWithShowHideValuesRemoved)
+                .put("dateOfIssue", time.now().toLocalDate().toString())
+                .put("standardDirectionOrder", Order.builder().orderStatus(SEALED).build())
+                .put("judgeAndLegalAdvisor", JudgeAndLegalAdvisor.builder().build())
+                .put("allocatedJudge", Judge.builder().build())
+                .put(HEARING_DETAILS_KEY, wrapElements(HearingBooking.builder()
+                    .startDate(LocalDateTime.of(2020, 10, 20, 11, 11, 11))
+                    .endDate(LocalDateTime.of(2020, 11, 20, 11, 11, 11))
+                    .venue("EXAMPLE")
+                    .build()))
+                .put("caseLocalAuthority", "example")
+                .put("dateSubmitted", time.now().toLocalDate().toString())
+                .build())
+            .build();
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToSubmitEvent(caseDetails);
+
+        CaseData caseData = mapper.convertValue(callbackResponse.getData(), CaseData.class);
+
+        List<Element<Direction>> localAuthorityDirections =
+            caseData.getStandardDirectionOrder().getDirections().stream()
+                .filter(direction -> direction.getValue().getAssignee() == LOCAL_AUTHORITY)
+                .collect(toList());
+
+        assertThat(localAuthorityDirections).isEqualTo(fullyPopulatedDirection);
+        AssertionsForClassTypes.assertThat(caseData.getStandardDirectionOrder().getOrderDoc()).isNotNull();
+        AssertionsForClassTypes.assertThat(caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor()).isNotNull();
+        AssertionsForClassTypes.assertThat(caseData.getJudgeAndLegalAdvisor()).isNull();
+    }
+
+    @Test
+    void aboutToSubmitShouldReturnErrorsWhenNoHearingDetailsExistsForSealedOrder() {
```

Comment:
inconsistency, you renamed 
aboutToSubmitShouldPopulateHiddenCCDFieldsInStandardDirectionOrderToPersistData -> shouldPopulateHiddenCCDFieldsInStandardDirectionOrderToPersistData
which is fine, but not other method names

Reply:
done